zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,409 @@
+            rsp.progress = task.downloadSize * 90 / task.expectedSize
+
+        if task.lastError is not None:
+            rsp.success = False
+            rsp.error = task.lastError
+        return jsonobject.dumps(rsp)
+
+    def complete_upload(task, ioctx):
+        # type: (UploadTask) -> None
+        try:
+            file_format = linux.get_img_fmt('rbd:' + task.tmpPath)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)